block-management/block-manager.scss

```diff
@@ -0,0 +1,20 @@
+.container {
```

Comment:
We should maybe start adding `/** @format */` to style files. And check/format them with prettier.

Reply:
Good idea! Done with 686c0a3.